### PR TITLE
PERFSCALE-2860 - Re-enable jobs on new schedule balancing

### PR DESCRIPTION
**Pull request by afcollins**
Restoring perfscale jobs on a renewed, balanced schedule.

### CODE REVIEW

**Comment by afcollins** [comment]
[Google sheet](https://docs.google.com/spreadsheets/d/1TPKUMzq6qOnDo_SNNJJ5k5ijtiWLKxHhcuwEEknoWdg/edit#gid=622986199&range=A1) generated from the configurations here.

**Comment by afcollins** [comment]
cc: @qiliRedHat @rpattath

**Comment by afcollins** [comment]
Alright, I rebalanced the jobs with those 4.16 that have been added since.

I also split apart monthly and bi-weekly jobs such as not to overburden any particular week-long Jedi shift.

We have 5 workloads  that all run on their own days, so each week contains roughly the same mix of each workload.

The "5 workloads" are: [ control-plane-24, control-plane-120, data-path, node-density-heavy, loaded-upgrade ]

The 'periodic' tab in [this gsheet](https://docs.google.com/spreadsheets/d/1TPKUMzq6qOnDo_SNNJJ5k5ijtiWLKxHhcuwEEknoWdg/edit#gid=1320254767&range=A1) is updated with the changes from this PR. Please review.

You may find visual schedules for:
* May 1st - 15th  : http://ocp-intlab-grafana.rdu2.scalelab.redhat.com/
* May 15th - 29th : http://ocp-intlab-grafana.rdu2.scalelab.redhat.com/crontab.next-2wks.html
* May 1st - 31st  : http://ocp-intlab-grafana.rdu2.scalelab.redhat.com/crontab.2024-may.html

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.13-nightly-x86-loaded-upgrade-from-4.12.yaml [hunk 1]
```diff
@@ -30,8 +30,8 @@ resources:
       cpu: 100m
       memory: 200Mi
 tests:
-- always_run: false
-  as: perfscale-aws-5nodes-loaded-upgrade-412to413-periodic
+- as: perfscale-aws-5nodes-loaded-upgrade-412to413-periodic
```
Comment (jtaleric) [inline ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.13-nightly-x86-loaded-upgrade-from-4.12.yaml:33]: should we just call this `5nodes-loaded-upgrade-412to413-periodic`, given the direction of some of the naming we have going on?

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.14-nightly-x86-loaded-upgrade-from-4.12.yaml [hunk 2]
```diff
@@ -37,8 +37,8 @@ resources:
       cpu: 100m
       memory: 200Mi
 tests:
-- always_run: false
-  as: perfscale-aws-5nodes-loaded-upgrade-412to413to414-periodic
+- as: perfscale-aws-5nodes-loaded-upgrade-412to413to414-periodic
```
Comment (jtaleric) [inline ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.14-nightly-x86-loaded-upgrade-from-4.12.yaml:40]: Same recommendation on the name

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.15-nightly-x86-loaded-upgrade-from-4.14.yaml [hunk 3]
```diff
@@ -30,8 +30,8 @@ resources:
       cpu: 100m
       memory: 200Mi
 tests:
-- always_run: false
-  as: perfscale-aws-24nodes-loaded-upgrade-414to415
+- as: perfscale-aws-24nodes-loaded-upgrade-414to415
```
Comment (jtaleric) [inline ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.15-nightly-x86-loaded-upgrade-from-4.14.yaml:33]: same.

**Review by jtaleric — COMMENTED** [review]
smallest nit ever -- very happy to see this moving forward! Thanks @afcollins!!

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.15-nightly-x86.yaml [hunk 4]
```diff
@@ -166,8 +166,8 @@ tests:
     - chain: openshift-qe-node-density-cni
     workflow: openshift-qe-installer-aws
   timeout: 2h0m0s
-- always_run: false
-  as: data-path-9nodes
+- as: data-path-9nodes
+  cron: 0 22 8,22 * *
```
Comment (qiliRedHat) [inline ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.15-nightly-x86.yaml:170]: @afcollins QE updated our profiles. This aws-ovn-fips is not needed anymore. The FIPs will be added to GCP. Let's move this to always_run: false.

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.15-nightly-x86.yaml [hunk 5]
```diff
@@ -232,8 +232,8 @@ tests:
     - chain: openshift-qe-node-density
     workflow: openshift-qe-installer-aws-ovn-fips
   timeout: 5h0m0s
-- always_run: false
-  as: node-density-24nodes-ovn-fips
+- as: node-density-24nodes-ovn-fips
+  cron: 0 8 9,23 * *
```
Comment (qiliRedHat) [inline ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.15-nightly-x86.yaml:236]: @afcollins Same reason as above. Pls move this to always_run: false.

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.16-nightly-x86.yaml [hunk 6]
```diff
@@ -216,7 +216,7 @@ tests:
     workflow: openshift-qe-installer-aws-ovn-fips
   timeout: 5h0m0s
 - as: node-density-24nodes-ovn-fips
-  cron: 0 9 * * 4
+  cron: 0 1 2,9,16,23 * *
```
Comment (qiliRedHat) [inline ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.16-nightly-x86.yaml:219]: @afcollins Same reason as above. Pls move this to always_run: false.

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.16-nightly-x86-loaded-upgrade-from-4.15.yaml [hunk 7]
```diff
@@ -31,7 +31,7 @@ resources:
       memory: 200Mi
 tests:
 - as: perfscale-aws-24nodes-loaded-upgrade-415to416
-  cron: 0 12 * * 1
+  cron: 0 0 3,10,17,24 * *
```
[the remark this answers is not in the document]
Comment (afcollins) [reply ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.16-nightly-x86-loaded-upgrade-from-4.15.yaml:34]: My idea with specifying days like this is to allow us to create a weekly/bi-weekly/monthly schedule. The day-of-week format `0 12 * * 1` means "run on Mondays", but I couldn't find any way to say "Run on every-other Monday".

By writing a weekly schedule like this, we can know the total number of jobs running on any given day/week/month.

"Month" i.e. 28 days and chop up from there.

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.16-nightly-x86.yaml (hunk 6 above)
Comment (afcollins) [reply ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.16-nightly-x86.yaml]: done

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.15-nightly-x86.yaml (hunk 5 above)
Comment (afcollins) [reply ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.15-nightly-x86.yaml]: done

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.15-nightly-x86.yaml (hunk 4 above)
Comment (afcollins) [reply ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.15-nightly-x86.yaml:170]: I don't see ovn-fips on the line you commented on , but I believe I moved all of them to always_run: false. Please check

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.16-nightly-x86-loaded-upgrade-from-4.15.yaml (hunk 7 above)
[the remark this answers is not in the document]
Comment (qiliRedHat) [reply ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.16-nightly-x86-loaded-upgrade-from-4.15.yaml:34]: I understood, I agree it's easier to understand by having a unified format.

**Pull request merged**